TypeCobol/Compiler/Nodes/Attributes.cs

```diff
@@ -420,22 +420,25 @@ internal class UnsafeAttribute: Attribute {
 
     internal class FunctionUserAttribute: Attribute {
 	    public object GetValue(object o, SymbolTable table) {
-		    var statement = ((Node)o).CodeElement as FunctionCaller;
-		    if (statement == null) return null;
-		    var functions = new List<FunctionCallInfo>();
-		
-		    var found = table.GetFunction(new URI(statement.FunctionCall.FunctionName));
-			
-		    if (found.Count > 1) throw new System.ArgumentException("Resolve ambiguity for "+found.Count+" items");
-		    var declaration = found[0];
-		    functions.Add(Create(statement.FunctionCall, declaration));
-		
-		    if (functions.Count == 0) return null;
-		    if (functions.Count == 1) return functions[0];
-		    return functions;
-	    }
+		    var statement = o as FunctionCaller;
+
+            string functionName = statement?.FunctionCall?.FunctionName;
+            if (functionName == null) return null;
+
+		    var found = table.GetFunction(new URI(functionName));
+            switch (found.Count)
+            {
+                case 0:
+                    return null;
+                case 1:
+                    return Create(statement.FunctionCall, found[0]);
```

Comment:
I've found this part in `skeleton.xml`:
https://github.com/TypeCobolTeam/TypeCobol/blob/67557bd7a7bd040183e45425addb7952365e168f/Codegen/src/Skeletons/skeletons.xml#L384-L422
and it looks like the expected return type is a single `FunctionCallInfo` instead of a list. Note the spaces after `function` in the variable declaration, maybe at some point we had cardinality here.

Reply:
Yes, maybe. 
This code is not used anymore. We can wait to see what happens with TypeCobol language to delete it.